public/css/nexthome.css

```diff
@@ -1,530 +1,649 @@
-        .decBlockchainWrapper .decredblockWrap .decredblock .info-block .expanded-block button {
-          background: none;
-          border: 1px solid #c3cad1;
-          width: 80%;
-          margin-left: 10%;
-          padding: 6px;
-          font-family: "Source Sans Pro", sans-serif;
-          border-radius: 4px;
-          margin-top: 8px; }
-        .decBlockchainWrapper .decredblockWrap .decredblock .info-block .expanded-block .data {
-          font-family: "Inconsolata", monospace;
-          font-size: 22px;
-          font-weight: bold;
-          display: flex;
-          flex-direction: row;
-          align-items: flex-end;
-          margin-left: 10px;
-          font-size: 16px;
-          line-height: 29px;
-          margin-left: 6px; }
-          .decBlockchainWrapper .decredblockWrap .decredblock .info-block .expanded-block .data p {
-            margin: 0px; }
-          .decBlockchainWrapper .decredblockWrap .decredblock .info-block .expanded-block .data span:first-child {
-          display: flex;
-          flex-direction: row;
-          align-items: flex-end; }
-        .decBlockchainWrapper .decredblockWrap .decredblock .info-block .expanded-block .subdata {
-          font-family: "Source Sans Pro", sans-serif;
-          font-size: 12px;
-          margin-left: 12%; }
-      .decBlockchainWrapper .decredblockWrap .decredblock .info-block img {
-        float: left; }
-    .decBlockchainWrapper .decredblockWrap .decredblock span.color-code {
-      position: absolute;
-      font-family: "Source Sans Pro", sans-serif;
-      font-size: 12px;
-      color: #2970FF;
-      font-weight: bold;
-      line-height: 35px;
-      left: 10px; }
-    .decBlockchainWrapper .decredblockWrap .decredblock span.timespan {
-      position: absolute;
-      font-family: "Source Sans Pro", sans-serif;
-      font-size: 12px;
-      color: #17173C;
-      line-height: 35px;
-      font-weight: bold;
-      right: 10px; }
-  .decBlockchainWrapper .decredblockWrap.expanded {
-    opacity: 1 !important; }
-    .decBlockchainWrapper .decredblockWrap.expanded .decredblock {
-      height: 192px;
-      top: -46px;
-      width: 90%;
-      left: 5%;
-      right: 5%;
-      z-index: 99; }
-      .decBlockchainWrapper .decredblockWrap.expanded .decredblock .info-block {
-        height: 180px; }
-
+    border: 1px solid #c4cbd2;
+    position: relative;
+    overflow: visible;
+}
+.decBlockchainWrapper .decredblockWrap .decredblock .block-rows {
+  display: flex;
+  flex-direction: column;
+  height: 90px;
+  padding: 1px;
+}
+.decBlockchainWrapper .decredblockWrap .decredblock .block-rows > * {
+  flex-basis: 5px;
+  flex-grow: 1;
+  width: 100%;
+  display: flex;
+  flex-wrap: nowrap;
+  align-items: stretch;
+  border-top: 1px solid white;
+}
+.decBlockchainWrapper .decredblockWrap .decredblock .block-rows > *:first-child {
+    border-top: none;
+}
+.decBlockchainWrapper .decredblockWrap .decredblock .block-rows > * > * {
```

Comment:
In general wildcard `*` selectors should be avoided in css. They are mostly just used for doing global resets to smooth out browser base-level styling/layout differences. (Eg https://www.paulirish.com/2012/box-sizing-border-box-ftw/ )

Also, I'm not a huge fan of long nested chains like this, since they are less performant and brittle in the face of markup changes. It's better to use a descriptive class name in the markup instead of a long nested chain of classes and dom traversal operators(the > are also not efficient because the browser can't reason about them from the bottom up). While css is smoking fast these days, css cruft builds up over time and still makes a noticeable difference in user experience on mobile and other resource constrained clients to keep an eye on performance best practices.

Being liberal with classes in your markup also makes it easier to understand and change things, since complexity is localized and not co-mingled with other elements.

Also, I would drop "dec" and "decred" from the classnames, I don't see how that adds any value, it's just dead weight.